modules/docs-tests/src/main/scala/sclicheck/sclicheck.scala

```diff
@@ -139,6 +139,8 @@ def mkBashScript(content: Seq[String]) =
      |${content.mkString("\n")}
      |""".stripMargin
 
+def removeAnsiColors(str: String) = str.replaceAll("\\e\\[[0-9]+m", "")
```

Comment:
it'd be neat if we had a shared `test-utils` module, which would allow for utils like this one to be shared with other tests (as this is a duplicate of `TestUtil.removeAnsiColors`). 
Just noting, we can refactor it some other time.